Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
+
+This guide is to help contributors write/modify E2E tests in webdriverIO during the hybrid state.
+
+## Selectors
+
+The `$` command is a short way to call the findElement command in order to fetch a single element on the page
+
+| Name                                     | Protractor                               |WebdriverIO           |
+| -----------------------------------------| -----------------------------------------|----------------------|
+| By class name                            |`element(by.css('.className'))`           |`$('.className')`     |
+| Element containing certain string Text   |`element(by.cssContainingText(tag, text))`|`$('tag=text')`       |
+| Multiple Elements                        |`element.all(by.css('.className'))`       |`$$('.className')`    |
+| By ID                                    |`element(by.css('#idName'))`              |`$('#idName')`        |
+| Chain Selectors                          |`element(by.css('some-css')).element(by.css('tag-within-css'))`  | `$('some-css').$('tag-within-css’)`      |
+| Selecting element one by one in loop     |`element.get(i)`                          |`$$(selector)[i]`     |
+
+There is no proper substitute available for the command `element.getWebElement()` in webdriverIO but we can use `browser.findElement()` as a substitute for this.
+
+## Actions
+
+| Actions                                  | Protractor                               |WebdriverIO            |
```

Comment:
"Actions" might be confusing given that we have "action.js"

Reply:
Updated.